style/src/slider.rs

```diff
@@ -5,11 +5,20 @@ use iced_core::Color;
 #[derive(Debug, Clone, Copy)]
 pub struct Appearance {
     /// The colors of the rail of the slider.
-    pub rail_colors: (Color, Color),
+    pub rail: Rail,
     /// The appearance of the [`Handle`] of the slider.
     pub handle: Handle,
 }
 
+/// The appearance of a slider rail
+#[derive(Debug, Clone, Copy)]
+pub struct Rail {
+    /// The colors of the rail of the slider.
+    pub rail_colors: (Color, Color),
+    /// The height of a slider rail and the width of a vertical slider.
+    pub rail_size: f32,
```

Comment:
I say we rename these: `colors` and `size` since it feels redundant to have it prefixed with `rail_`